build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/release/ReleaseToolsPlugin.java

```diff
@@ -67,10 +69,12 @@ public void apply(Project project) {
                 task.dependsOn(validateChangelogsAgainstYamlTask);
             });
 
-        project.getTasks().register("generateReleaseNotes", GenerateReleaseNotesTask.class).configure(task -> {
+        final Function<Boolean, Action<GenerateReleaseNotesTask>> configureGenerateTask = shouldConfigureYamlFiles -> task -> {
             task.setGroup("Documentation");
             task.setDescription("Generates release notes from changelog files held in this checkout");
```

Comment:
I would expect different descriptions for both task making clear what they differ in

Reply:
Ah, good catch, I've changed that.